odra-core/src/host_env.rs

```diff
@@ -18,6 +18,10 @@ impl HostEnv {
         HostEnv { backend }
     }
 
+    pub fn error(&self) -> Option<OdraError> {
```

Comment:
Why do we need this function?

Reply:
It had already been there when I took it over. But having `try_*` functions and `env.last_call_result()` soon, I think we don't need it,